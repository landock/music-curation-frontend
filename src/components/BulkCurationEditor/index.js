import React, { PureComponent } from 'react';
import PropTypes from 'prop-types';
import { Form, Grid } from 'semantic-ui-react';

class BulkCurationEditor extends PureComponent {
  render() {
    return (
      <Grid.Row>
        <Grid.Column textAlign="left">
          <Form widths="equal">
            <Form.Group inline>
              <Form.Field inline>
                <Form.Checkbox label="Select All" />
              </Form.Field>
              <Form.Select fluid />

              <Form.Select fluid />
              <Form.Button floated="right" color="red">
                New Curation
              </Form.Button>
            </Form.Group>
          </Form>
        </Grid.Column>
      </Grid.Row>
    );
  }
}

BulkCurationEditor.propTypes = {};

export default BulkCurationEditor;
