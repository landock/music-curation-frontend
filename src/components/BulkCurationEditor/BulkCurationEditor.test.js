
import React from 'react'
import { shallow, mount, render } from 'enzyme';
import renderer from 'react-test-renderer'
import BulkCurationEditor from '.'

it('BulkCurationEditor: default', () => {
  const component = renderer.create(<BulkCurationEditor />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})