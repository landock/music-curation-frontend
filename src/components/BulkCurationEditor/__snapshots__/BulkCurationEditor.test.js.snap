// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BulkCurationEditor: default 1`] = `
<form
  action={undefined}
  className="ui equal width form"
  onSubmit={[Function]}
>
  <div
    className="inline fields"
  >
    <div
      className="inline field"
    >
      <div
        className="field"
      >
        <div
          className="ui checkbox"
          content={undefined}
          onChange={[Function]}
          onClick={[Function]}
          onMouseDown={[Function]}
        >
          <input
            checked={false}
            className="hidden"
            id={undefined}
            name={undefined}
            onClick={[Function]}
            readOnly={true}
            required={undefined}
            tabIndex={0}
            type="checkbox"
            value={undefined}
          />
          <label
            htmlFor={undefined}
          >
            Select All
          </label>
        </div>
      </div>
    </div>
    <div
      className="field"
    >
      <div
        aria-busy={undefined}
        aria-disabled={undefined}
        aria-expanded={false}
        aria-multiselectable={undefined}
        className="ui fluid selection dropdown"
        content={undefined}
        onBlur={[Function]}
        onChange={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onMouseDown={[Function]}
        required={undefined}
        role="listbox"
        tabIndex={0}
        type={undefined}
      >
        <div
          aria-live="polite"
          className="default text"
          role="alert"
        >
          Actions
        </div>
        <i
          aria-hidden="true"
          className="dropdown icon"
          onClick={[Function]}
        />
        <div
          className="menu transition"
        >
          <div
            aria-checked={false}
            aria-disabled={undefined}
            aria-selected={true}
            className="selected item"
            onClick={[Function]}
            role="option"
            style={
              Object {
                "pointerEvents": "all",
              }
            }
          >
            <span
              className="text"
            >
              Afghanistan
            </span>
          </div>
        </div>
      </div>
    </div>
    <div
      className="field"
    >
      <div
        aria-busy={undefined}
        aria-disabled={undefined}
        aria-expanded={false}
        aria-multiselectable={undefined}
        className="ui fluid selection dropdown"
        content={undefined}
        onBlur={[Function]}
        onChange={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onMouseDown={[Function]}
        required={undefined}
        role="listbox"
        tabIndex={0}
        type={undefined}
      >
        <div
          aria-live="polite"
          className="default text"
          role="alert"
        >
          Add to
        </div>
        <i
          aria-hidden="true"
          className="dropdown icon"
          onClick={[Function]}
        />
        <div
          className="menu transition"
        >
          <div
            aria-checked={false}
            aria-disabled={undefined}
            aria-selected={true}
            className="selected item"
            onClick={[Function]}
            role="option"
            style={
              Object {
                "pointerEvents": "all",
              }
            }
          >
            <span
              className="text"
            >
              Afghanistan
            </span>
          </div>
        </div>
      </div>
    </div>
    <div
      className="field"
    >
      <button
        className="ui red right floated button"
        disabled={undefined}
        onClick={[Function]}
        required={undefined}
        role="button"
        tabIndex={undefined}
        type={undefined}
      >
        New Curation
      </button>
    </div>
  </div>
</form>
`;
